test/config/ytt/ingress/kourier/kourier-replicas.yaml

```diff
@@ -1,15 +1,7 @@
 #@ load("@ytt:overlay", "overlay")
 #@ load("helpers.lib.yaml", "subset")
 
-#@ def net_kourier_controller():
-#@  names=[
-#@    subset(name="net-kourier-controller"),
-#@    subset(name="3scale-kourier-gateway"),
-#@  ]
-#@  return overlay.and_op(subset(kind="Deployment"), overlay.or_op(*names))
-#@ end
-
-#@overlay/match by=net_kourier_controller()
+#@overlay/match by=subset(kind="Deployment", name="3scale-kourier-gateway")
```

Comment:
Out of curiosity is there a reason why you don't want to run the controller in an HA setting? Does it not affect the scale tests?

Reply:
actually nm - I see it was always the gateway that was scaled

https://github.com/knative/serving/blob/release-0.20/test/e2e-networking-library.sh#L109-L110

Reply:
well that's a different discussion, is it? This matcher was failing because it **did** match 2 deployments and by the naming of the function (`net_kourier_controller`) and by looking at the git history, I figured this is supposed to only match the gateway.

Reply:
> well that's a different discussion, is it? 

Yup